### ISSUE TITLE
[Feature]: default global config

### ISSUE THREAD

**Issue by itsactuallyezra** [LordDeatHunter/FabricWaystones#161]
### Description

A simple config to allow you to set the waystones global by default.

### Describe the solution you would like

It would be amazing to have that config, and be able to lock the toggle, so every waystone is always global.

**Comment by allanonmage** [comment]
Yes, the public/private part of waystones is not explained well so my server figured you'd have to go there manually first before being able to use a waystone.

**Comment by lihaan** [comment]
Issue seems inactive, but I would like to show my support for this feature. It is useful for small-scale SMPs among groups of friends, where it doesn't make sense for everyone to have to travel via their own means to a certain waystone in order to activate it. Since cooperative play is actively encouraged on such servers, it would be great wfor waystones to have an auto-global option so that players will not forget to enable it for the rest of their friends.
